js/gdm/loginDialog.js

```diff
@@ -1179,6 +1179,7 @@ const LoginDialog = new Lang.Class({
                              this._promptEntry.reactive = false;
                              return this._beginVerificationForUser(userName);
                          }
+                         return undefined;
```

Comment:
Do we really need to return something in this function? Couldn't we simply drop the other return when calling `_beginVerificationForUser()`? I think the return value is not being used anyway...